src/main.cpp

```diff
@@ -2,6 +2,10 @@
 
 #include <QApplication>
 
+#ifdef INCLUDE_GIT_COMMIT_HASH
```

Comment:
`INCLUDE_GIT_COMMIT_HASH` sa vobec nastavuje v CMakeLists.txt ???

Reply:
_option(INCLUDE_GIT_COMMIT_HASH "Embed the latest git commit abbreviated hash into the sources." ON)_ je v Git.cmake